assets/stylesheets/kitten/helpers/typography/_font-size.scss

```diff
@@ -62,10 +63,15 @@
     $line-height: ceil($font-size / $base-line-height) * $base-line-height;
   }
 
+  $is-important: null;
+  @if $important {
+    $is-important: !important;
+  };
```

Comment:
Est-ce que la ternaire Sass marche `$is-important: if($important, !important, null);` ?